lib/vdc/BOVCollection.cpp

```diff
@@ -60,8 +66,8 @@ const std::string BOVCollection::_doubleFormatString = "DOUBLE";
 
 BOVCollection::BOVCollection()
 : _time(_defaultTime), _dataFile(_defaultFile), _dataFormat(_defaultFormat), _variable(_defaultVar), _dataEndian(_defaultEndian), _centering(_defaultCentering), _byteOffset(_defaultByteOffset),
```

Comment:
I thought endian stuff is removed?

Reply:
I'm still finding unused tokens, in case we want to implement them later.

Reply:
Isn't it additional 1) bug potentials and 2) complexity (imagine someone else debugging) without any benefit?

Reply:
Regarding 1)
No, because the parsing code is the same for all tokens.  When I started this, I wrote the code to parse all BOV tokens, and reduced the tokens that we supported as time allowed.  If we want to extend the tokens that are supported, we need to parse these anyways.

Regarding 2)
It's really just one line of code.  I don't think it adds complexity.  If a user adds a bad value for an unused token, yes this will throw an error.  But it's one line per token, and if we're following VisIt's example, I'd like to parse all of the tokens for completeness.  Even if they're not all used.

Reply:
Ok, how about this: 
1) document in the code in the clearest possible language that these are dummy values being passed around without being used by anybody.
2) document in the code in the clearest possible language that what will happen when a good value is passed in, when a bad value is passed in, when a duplicate value is passed in, etc.

The goal is, for example, 6 months later Sam is tasked to complete the current BOV reader to support all tokens, Sam can know the nuances that you described in the past month in multiple PR threads. (Possibly: rename all the unused tokens/handles etc. with a prefix/postfix `dummy`.)

Reply:
> 1. document in the code in the clearest possible language that these are dummy values being passed around without being used by anybody.
> 2. document in the code in the clearest possible language that what will happen when a good value is passed in, when a bad value is passed in, when a duplicate value is passed in, etc.

I've added additional comments in the BOVCollection.h/cpp files wherever these variables are defined or used.

I've also added more information to DCBOV.h, where you can find the rules for things like what happens when a bad value is passed in for different types of tokens, and an example BOV header file.

> How big the effort is to add them back in, and how big is the chance that they will be added back in?

I think that's hard to predict.  John has talked about how wacky these binary data files can get, and I think that VisIt wouldn't have these tokens without good reason.

The unused values haven't resulted in any bugs, and they comply with the specification I was given.  Removing them, and then re-adding them later would be more effort than letting things be.